examples/test/azureml/test_bert_ptq_cpu_aml.py

```diff
@@ -6,7 +6,7 @@
 
 import pytest
 
-from ..utils import check_output, get_example_dir, patch_config
+from examples.test.utils import check_output, get_example_dir, patch_config
```

Comment:
does this import work if we want to run an example locally from a different directory that is not the repo root? like from `examples/test` itself?

Reply:
If that works, can you also remove the TID252 rule from here https://github.com/microsoft/Olive/blob/798508b4d31d76362671d0c56b9bbce6b764a024/pyproject.toml#L193?

Reply:
It did not work.

Just remove the checker in pre-commit config.